lnd_test.go

```diff
@@ -3644,6 +3645,704 @@ func testMultiHopPayments(net *lntest.NetworkHarness, t *harnessTest) {
 	closeChannelAndAssert(ctxt, t, net, carol, chanPointCarol, false)
 }
 
+func testSpiderShortestPath(net *lntest.NetworkHarness, t *harnessTest) {
```

Comment:
Can you add a comment here detailing what this test is about similar to all the above tests?